src/modules/app.js

```diff
@@ -0,0 +1,30 @@
+import post from './post.js';
+import get from './get.js';
+import addScore from './addScore.js';
+
+const submitBtn = document.querySelector('.submit-Btn');
+const removeBtn = document.querySelector('.remove-Btn');
+const userName = document.querySelector('.text-input');
+const userScore = document.querySelector('.score-input');
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const user = userName.value;
+  const score = userScore.value;
+  if (user && score) {
+    post({ user, score });
+  }
+});
+
+removeBtn.addEventListener('click', async () => {
```

Comment:
- [ ] Kindly ensure the user can only view the added score after refreshing the page not automatically, I also noticed when hitting the refresh button multiple times it readded already added score in the list, kindly check on it.